packages/utils/src/lib/reports/utils.ts

```diff
@@ -16,12 +16,23 @@ import {
   readTextFile,
 } from '../file-system';
 import { SCORE_COLOR_RANGE } from './constants';
+import { image, style } from './md';
 import { ScoredReport, SortableAuditReport, SortableGroup } from './types';
 
 export function formatReportScore(score: number): string {
   return Math.round(score * 100).toString();
 }
 
+export function formatScoreWithColor(
```

Comment:
Where it makes sense, it would be nice to cover helper functions with unit tests (including those helper functions in the generate file itself.

Reply:
Added some more unit tests.